tests/monitors/docker_container_stats/docker_container_stats_test.py

```diff
@@ -5,10 +5,15 @@
 
 from tests.helpers.agent import Agent
 from tests.helpers.assertions import has_datapoint_with_dim, has_datapoint_with_metric_name
+from tests.helpers.metadata import Metadata
 from tests.helpers.util import ensure_always, run_service, wait_for
+from tests.helpers.verify import verify_included_metrics, verify_all_metrics
 
 pytestmark = [pytest.mark.docker_container_stats, pytest.mark.monitor_without_endpoints]
 
+METADATA = Metadata.from_package("docker")
+METADATA.all_metrics = METADATA.all_metrics - {"memory.stats.swap"}
```

Comment:
I don't think it's a good idea to change `METADATA.all_metrics` as there's also `METADATA.nonincluded_metrics` which won't be correct (even if it might not matter for this particular test). I'd just set it to a different variable and instead of calling `verify_all_metrics` call `verify` and pass that variable to verify.